ci_framework/roles/hive/tasks/openstack_claim_cluster.yml

```diff
@@ -0,0 +1,75 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+# Claims a cluster from a the provided pool
+- name: Claim an OpenShift Cluster
+  vars:
+    claim_name: "{{ cifmw_hive_openstack_claim_name }}"
+    cluster_pool: "{{ cifmw_hive_openstack_pool_name }}"
+    rtn_kube: "{{ cifmw_hive_artifacts_dir }}"
+    life_time: "{{ cifmw_hive_lifetime | default(false) }}"
+    namespace: "{{ cifmw_hive_namespace }}"
+
+  block:
+    - name: Check if ClusterPool exists
```

Comment:
Use `ansible.builtin.command` instead if no shell capabilities needed. Applies to all shell usages in the role.

Reply:
ack, done